up42/storage.py

```diff
@@ -0,0 +1,91 @@
+from typing import Dict, List, Union
+
+from tqdm import tqdm
+
+from up42.auth import Auth
+from up42.order import Order
+from up42.asset import Asset
+from up42.utils import get_logger
+from up42.tools import Tools
+
+logger = get_logger(__name__)
+
+
+class Storage(Tools):
+    def __init__(self, auth: Auth):
+        """
+        The Storage class can query all available assets within an UP42 workspace.
+        """
+        self.auth = auth
+        self.workspace_id = auth.workspace_id
+
+    def __repr__(self):
+        env = ", env: dev" if self.auth.env == "dev" else ""
+        return f"Storage(workspace_id: {self.workspace_id}{env})"
+
+    def _paginate(self, url: str) -> List[Dict]:
+        """
+        Helper to fetch list of items in paginated endpoint.
+
+        Args:
+            url (str): The base paginated endpoint.
+
+        Returns:
+            List[Dict]: List of all paginated items.
+        """
+        first_pagination_response = self.auth._request(request_type="GET", url=url)
+        output = first_pagination_response["data"]["content"]
```

Comment:
Is output already a list here so that you can append to it further below? Looks like it would be a dict here?

Reply:
Yup, this is already a list - since pagination implies that "content" will include always a list of things.